tools/cosmovisor/go.mod

```diff
@@ -3,6 +3,7 @@ module cosmossdk.io/tools/cosmovisor
 go 1.19
 
 require (
+	cosmossdk.io/x/upgrade v0.0.0-20230127052425-54c8e1568335
```

Comment:
Can you completely revert cosmovisor to the state it has on main? It is depending on rc1, so it should not have any change in this PR.